ginga/rv/plugins/Preferences.py

```diff
@@ -60,15 +60,44 @@
 used.  Click the control to show the list, or simply scroll the mouse
 wheel while hovering the cursor over the control.
 
+.. note:: Ginga comes with a good selection of color maps, but should you
+          want more, you can add custom ones or, if ``matplotlib`` is
+          installed, you can load all the ones that it has.
+          See "Customizing Ginga" for details.
+
 The "Intensity" control selects which intensity map should be used
 with the color map.  The intensity map is applied just before the color
 map, and can be used to change the standard linear scale of values into
 an inverted scale, logarithmic, etc.
 
-Ginga comes with a good selection of color maps, but should you want
-more, you can add custom ones or, if ``matplotlib`` is installed, you
-can load all the ones that it has.
-See "Customizing Ginga" for details.
+The "Invert CMap" checkbox can be used to invert the selected color map
+(note that a number of colormaps are also selectable from the "Colormap"
+controlin inverted form).
+
+The "Rotate" control can be used to rotate the colormap, while the
+"Unrotate CMap" button will restore the rotation to its default, unrotated
+state.
+
+The "Color Defaults" button will reset all the color mapping controls to
```

Comment:
Personally, I don't see much use in "Color Defaults" because I can already "uninvert" and "unrotate" separately. But I guess does not hurt. 🤷